tap_facebook_pages/streams.py

```diff
@@ -172,7 +172,10 @@ def request_records(self, context: Optional[dict]) -> Iterable[dict]:
             select posts.video.id, created_at 
             from `nymag-analytics-157315.organic_social.dim__facebook_posts` posts
             left join video_tables on posts.video.id = video_tables.id
-            where video_tables.id is null
+            where 
```

Comment:
just to make sure i understand, your query doesn't need to actually select the `page_id` because in `get_child_context()` for this stream you're using `context["page_id"]` and not `record["page_id"]`?